### PR TITLE
Track unmaintained repo - mailru/easyjson

### PR DESCRIPTION
**Pull request by dims**
#### What type of PR is this?

Related to https://github.com/kubernetes/kubernetes/issues/117553 - this library seems unmaintained
/kind cleanup

#### Which issue(s) this PR fixes:

Fixes #

#### Does this PR introduce a user-facing change?

```release-note
NONE
```

#### Additional documentation e.g., KEPs (Kubernetes Enhancement Proposals), usage docs, etc.:

```docs

```

/sig architecture
/area code-organization

### CODE REVIEW

**Review by pacoxu — APPROVED** [review]

File: hack/unwanted-dependencies.json [hunk 1]
```diff
@@ -47,6 +47,7 @@
       "github.com/hashicorp/serf": "MPL license not in CNCF allowlist",
       "github.com/influxdata/influxdb1-client": "db/datastore clients should not be required",
       "github.com/json-iterator/go": "refer to #105030",
+      "github.com/mailru/easyjson": "refer to #117553",
```
Comment (liggitt) [inline hack/unwanted-dependencies.json:50]: if the reason we want to add this is that it is unmaintained, that could be ok (though it's not clear to me the repo is in archive mode), though I would just say that here.

the primary complaint in the referenced issue is not about active maintenance, it's about the maintainer, which I'm not sure I agree with, given https://github.com/kubernetes/kubernetes/issues/117553#issuecomment-1522452761

Comment (dims) [reply hack/unwanted-dependencies.json]: Updated to say `unmaintained`

**Pull request merged**